runelite-client/src/main/java/net/runelite/client/plugins/grotesqueguardians/GrotesqueGuardiansConfig.java

```diff
@@ -0,0 +1,87 @@
+package net.runelite.client.plugins.grotesqueguardians;
+
+import java.awt.Color;
+import net.runelite.client.config.Config;
+import net.runelite.client.config.ConfigGroup;
+import net.runelite.client.config.ConfigItem;
+
+@ConfigGroup("grotesqueguardians")
+public interface GrotesqueGuardiansConfig extends Config
+{
+	@ConfigItem(
+		keyName = "displayTextOverlays",
+		name = "Display Text Overlays",
+		description = "Configures the display of text overlays",
+		position = 1
+	)
+	default boolean displayTextOverlays()
+	{
+		return true;
+	}
+
+	@ConfigItem(
+		keyName = "displayLightning",
+		name = "Display Lightning Tiles",
+		description = "Configures the display of lightning special attack tiles",
+		position = 2
+	)
+	default boolean displayLightning()
+	{
+		return true;
+	}
+
+	@ConfigItem(
+		keyName = "lightningColor",
+		name = "Lightning Tile Color",
+		description = "Configures the color of the lightning special attack tiles",
+		position = 3
+	)
+	default Color lightningColor()
```

Comment:
I feel like all these color configurations are unnecessary

Reply:
Color preferences allow users to easily identify which GraphicObject is appearing, rather than being a single color across the board with text to examine which is being spawned.

Reply:
See my previous reply